sources/main.ts

```diff
@@ -96,7 +97,7 @@ query openPullRequests($owner: String!, $repo: String!, $after: String) {
 	if (pullRequests.length === 0) {
 		return;
 	}
-
+	let dirtyStatuses: any = {};
```

Comment:
Does 
```suggestion
	let dirtyStatuses: Record<number, boolean> = {};
```
work as well? I'd like to avoid `any` if we have such a small alternative that is type-safe.